docs/source/ttnn/ttnn/demos.rst

```diff
@@ -0,0 +1,33 @@
+Building and Uplifting Demos
```

Comment:
This looks great! We should broadcast this to #tt-metal-developers once it is in.